src/compat_check.jl

```diff
@@ -0,0 +1,167 @@
+
+"""
+    compatible(current::VersionNumber, desired::VersionNumber)
+    compatible(current::VersionNumber, desired::String)
+    compatible(current::VersionNumber, desired::Pkg.Types.VersionSpec)
+
+Check whether `current` version is compatible with `desired`.
+"""
+compatible(current::VersionNumber, desired::VersionNumber) = compatible(current, string(desired))
```

Comment:
Is this dispatch needed?

Reply:
Seems to be hit in code coverage but will confirm